x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.test.tsx

```diff
@@ -1024,7 +1024,7 @@ describe('<CspPolicyTemplateForm />', () => {
     it(`project ID is required for Manual users`, () => {
       let policy = getMockPolicyGCP();
       policy = getPosturePolicy(policy, CLOUDBEAT_GCP, {
-        project_id: { value: undefined },
+        'gcp.project_id': { value: undefined },
```

Comment:
yes, this is the rename of the vars 
to answer your 2nd question I'm not 100% sure 🤔 
@kfirpeled 👀

Reply:
Yea, also, if we are changing the names of these vars what happens if you save a GCP integration (without the changes in the PR) and then switch to using the code changes here. Is kibana going to point to a missing value in that case?

Reply:
to answer the above question, yes i believe so as that value name has been changed by this PR

Reply:
So the question is if Kibana 8.10 supports integrations 1.5.0, 1.5.1 and 1.5.2 as described in the [changelog](https://github.com/elastic/integrations/blob/main/packages/cloud_security_posture/changelog.yml)?

```yml
- version: "1.5.2-preview2"
  changes:
    - description: Refactor GCP credentials - Bump Version
      type: enhancement
      link: https://github.com/elastic/integrations/pull/7546
- version: "1.5.2-preview1"
  changes:
    - description: Refactor GCP credentials
      type: enhancement
      link: https://github.com/elastic/integrations/pull/1234
- version: "1.5.1"
  changes:
    - description: Bump version to 1.5.1
      type: enhancement
      link: https://github.com/elastic/integrations/pull/7493
    - description: Remove capitalization and change type for tags
      type: bugfix
      link: https://github.com/elastic/integrations/pull/7485
- version: "1.5.0"
```

If these changes go backported to `Kibana 8.10`, it means the Kibana UI won't support 1.5.0 and 1.5.1 properly for GCP, so I think we should update the MIN GCP version [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/cloud_security_posture/public/common/constants.ts#L217) to `1.5.2`?

```
export const MIN_VERSION_GCP_CIS = '1.5.0';
```

Reply:
Can you explain what upping the min version does? Looking at the code it seems like it will mark the policy invalid when a user goes to edit it. Are we missing a migration process here?

Reply:
https://github.com/elastic/kibana/blob/main/x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/gcp_credential_form.tsx#L352

Reply:
e.g. something like this? 
https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/saved_objects/migrations/security_solution/to_v8_8_0.ts

I've never had to migrate a savedobject, so not sure how this gets triggered and when.

Reply:
> Can you explain what upping the min version does? Looking at the code it seems like it will mark the policy invalid when a user goes to edit it. Are we missing a migration process here?

GCP Min version It's a UI feature flag, it doesn't allow the user to save a GCP integration and shows that the version is not compatible, it doesn't render the manual and Google shell options for GCP but the code doesn't break.

<img width="1478" alt="image" src="https://github.com/elastic/kibana/assets/19270322/d5133190-d0f8-4e0c-8649-4ef6d157c844">

Reply:
> Can you explain what upping the min version does? Looking at the code it seems like it will mark the policy invalid when a user goes to edit it. Are we missing a migration process here?

I suggest we don't perform any migration here, as it can be avoided since there's no old data to migrate from, Users with Kibana 8.9 are currently not able to install GCP integrations, only on Kibana 8.10 which will be released soon, but instead of going to any migration process I suggest we prevent the installation with the incompatible version.

Reply:
Oh good, yea sorry, I wasn't aware this feature wasn't released yet.